coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,300 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    This aspect checks on the format of your code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, uncorrect indentation etc...
+
+        x = {  'a':37,'b':42,
+
+        'c':927}
+
+        y = 'hello ''world'
+        z = 'hello '+'world'
+        a = 'hello {}'.format('world')
+        class foo  (     object  ):
+            def f    (self   ):
+                return       37*-+2
+            def g(self, x,y=42):
+                return y
+        def f  (   a ) :
+            return      37+-+a[42-x :  y**3]
+        """
+        example_language = 'python'
+        importance_reason = """
+        Your coding style matters, it can drastically affects the readability,
+        and maintainability of your program and might as well introduce bugs.
+        """
+        fix_suggestions = """
+        Defining a clearly and thoughful coding style (based on the available
+        ones given the programming language in use) and strictly respect it or
+        apply it through out the implementation of your project.
+        """
+
+
+@Formatting.subaspect
+class Length:
+    """
+    This aspect checks on each line's length and the number of lines
```

Comment:
again, the aspect doesn't really do a thing. Also the Length aspect isn't specific to the line right? That's LineLength.